lib/honeybadger/splitproject.rb

```diff
@@ -1,10 +1,22 @@
 require 'honeybadger/honeybadger'
 require 'honeybadger/key_helpers'
+require 'honeybadger/team_filter'
 
 module Honeybadger
   module Splitproject
-    
+    @@honeybadger_filters = [TeamFilter.new]
+
     class << self
+
+      def filters
+        @@honeybadger_filters
+      end
+
+      def add_filter(klass)
+        raise "MissingFilterMethod" unless klass.respond_to?(:filter)
```

Comment:
extending a class doesn't really help the sub class override `.filter` with the same contract in ruby.

We have code in malibu to attach the redshift instance to honeybadger notifications.  Making a system for adding such features here will avoid the order dependency if both were trying to alias_method, since this one is order dependent.